problems/removeNumbersAtOddIndices.js

```diff
@@ -2,8 +2,15 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+const removeNumbersAtOddIndices = (arr) => {
+    let output = []
+    for(let i = 0; i < arr.length; i++){
+        if(i % 2 === 1){
```

Comment:
It looks like this will keep the numbers at odd indicies. Probably want to keep the even ones instead.